Cargo.toml

```diff
@@ -9,17 +9,14 @@ repository = "https://github.com/RReverser/wasmbin"
 categories = ["wasm", "parser-implementations"]
 keywords = ["webassembly", "wasm", "parser", "serializer"]
 
-exclude = [
-    "tests/testsuite",
-    "benches/fixture.wasm",
-]
+exclude = ["tests/testsuite", "benches/fixture.wasm"]
 
 [dependencies]
 leb128 = "0.2.5"
-thiserror = "1.0.56"
+thiserror = "2.0"
 wasmbin-derive = { version = "0.2.3", path = "derive" }
-custom_debug = "0.6.0"
-once_cell = "1.19.0"
+custom_debug = "0.6"
+once_cell = "1.21"
```

Comment:
I think we can get rid of `once_cell` dependency altogether nowadays - `std::sync::OnceLock` now exists and is suitable for our needs.